### PR TITLE
AT-116: Setup translation-service

### PR DESCRIPTION
**Pull request by sprak3000**
Update `hello-go` based on conversation with DevOps to bring it inline with expectations for base service templates.

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# coding styles between different editors and IDEs
+# http://editorconfig.org
+
+root = true
+
+[*]
+
+# Change these settings to your own preference
+indent_style = tab
+indent_size = 2
```
Comment (sprak3000) [inline .editorconfig:12]: Where Go is opinionated and has its own formatting tool, may be some other way of enforcing style via the tool.

File: helm/serviceName/.helmignore [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
```
Comment (sprak3000) [inline helm/serviceName/.helmignore:1]: Unfamiliar with Helm and how it relates to our services. Copied this over from `hello-nest` and modified things best as I could.

**Comment by sprak3000** [comment]
Not sure if the JIRA lint should be failing here or not...

File: .editorconfig (hunk 1 above)
Comment (blairham) [reply .editorconfig:12]: That is only for *.go files not all. Also, we should be using spaces. If you want to format .go files differently you can do the follow:

# File type overrides
[*.go]
indent_size = tab

File: helm/serviceName/.helmignore (hunk 2 above)
Comment (blairham) [reply helm/serviceName/.helmignore:1]: helm is like an npm package but for Kubernetes

**Review by blairham — CHANGES_REQUESTED** [review]
I would just remove the helm stuff, it might be similar but should be obtained from your other go project, not the node stuff and we'll most likely need to make some adjustments there too.

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+          name: Publish <serviceName>-service image with commit hash tag
+          command: docker push hqo-docker.jfrog.io/<serviceName>-service:$CIRCLE_SHA1
+      - when: # master branch
+          condition:
+            equal: [master, << pipeline.git.branch >>]
+          steps:
+            - run:
+                name: Publish latest <serviceName>-service image
+                command: docker push hqo-docker.jfrog.io/<serviceName>-service:latest
+  update_helm_chart:
```
Comment (blairham) [inline .circleci/config.yml:59]: You will need to delete this section as you don't have any helm right now

File: .circleci/config.yml [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+      - validate_scripts
+      - run-unit-tests
+      - build_and_push_image:
+          context:
+            - artifactory
+            - github
+          requires:
+            - validate_scripts
+            - run-unit-tests
+      - update_helm_chart:
```
Comment (blairham) [inline .circleci/config.yml:118]: Remove the workflow too, same reason as the job

**Review by blairham — CHANGES_REQUESTED** [review]
one more removal for helm and should be g2g

**Review by blairham — APPROVED** [review]

**Review by SirSamuelJoseph — APPROVED** [review]

**Pull request merged**